bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -480,11 +485,6 @@ crate::internal_macros::define_extension_trait! {
             InstructionIndices::from_instructions(self.instructions_minimal())
         }
 
-        /// Writes the human-readable assembly representation of the script to the formatter.
-        fn fmt_asm(&self, f: &mut dyn fmt::Write) -> fmt::Result {
-            bytes_to_asm_fmt(self.as_ref(), f)
-        }
```

Comment:
This is (correctly) considered breaking by cargo-semver-checks and it's OK to change it. So I think the only issue in the PR are missing `pub use` items.

Reply:
> This is (correctly) considered breaking by cargo-semver-checks and it's OK to change it. 

This is another instance of breaking in relation to `master` but not in relation to whats released, right?

> So I think the only issue in the PR are missing pub use items.

I'm not sure right now what pub use items you mean.

Reply:
The specific edit un-breaks the release in this one method but the overall change is breaking because of extension traits. Not a big deal though.

> I'm not sure right now what pub use items you mean.

The tool says some paths are no longer accessible, I guess you've removed them by accident.

Reply:
I don't know whats up with the tool

> struct bitcoin::blockdata::script::ScriptBuf, previously in file bitcoin/src/blockdata/script/owned.rs:27

IIUC it says `ScriptBuf` is not available but we have a public re-export 
```rust
#[rustfmt::skip]                // Keep public re-exports separate.
#[doc(inline)]
pub use primitives::script::ScriptBuf;
```

And also a wildcard re-export in `script/mod.rs`.

Reply:
We need to check if this is one of the known limitations of the tool.